modin/core/execution/ray/implementations/pandas_on_ray/io/io.py

```diff
@@ -237,9 +237,7 @@ def func(df, **kw):
             max_retries=0,
         )
         # pending completion
-        RayWrapper.materialize(
-            [partition.list_of_blocks[0] for partition in result.flatten()]
-        )
+        qc._modin_frame._partition_mgr_cls.get_objects_from_partitions(result.flatten())
```

Comment:
This is mostly a refactoring change, but it's also safer to use since the function triggers the materialization via `force_materialization`.